batch_processing.sh

```diff
@@ -117,8 +117,8 @@ sct_warp_template -d t2s.nii.gz -w warp_template2t2s.nii.gz
 # Subtract GM segmentation from cord segmentation to obtain WM segmentation
 sct_maths -i t2s_seg.nii.gz -sub t2s_gmseg.nii.gz -o t2s_wmseg.nii.gz
 # Compute cross-sectional area of the gray and white matter between C2 and C5
-sct_process_segmentation -i t2s_wmseg.nii.gz -vert 2:5 -perlevel 1 -o csa_wm.csv
-sct_process_segmentation -i t2s_gmseg.nii.gz -vert 2:5 -perlevel 1 -o csa_gm.csv
+sct_process_segmentation -i t2s_wmseg.nii.gz -vert 2:5 -perlevel 1 -o csa_wm.csv -angle-corr-centerline t2s_seg.nii.gz
+sct_process_segmentation -i t2s_gmseg.nii.gz -vert 2:5 -perlevel 1 -o csa_gm.csv -angle-corr-centerline t2s_seg.nii.gz
```

Comment:
Naturally, adding angle correction here changed the values for batch_processing.sh:

```
 =========================== short test summary info ============================
FAILED testing/batch_processing/test_batch_processing.py::test_batch_processing_results[t2s/csa_gm.csv-3-MEAN(area)] - assert 12.515230379832953 == 12.487533885581323 ± 1.2e-05
  
  comparison failed
  Obtained: 12.515230379832953
  Expected: 12.487533885581323 ± 1.2e-05
FAILED testing/batch_processing/test_batch_processing.py::test_batch_processing_results[t2s/csa_wm.csv-3-MEAN(area)] - assert 64.93314479093739 == 64.93464311280606 ± 6.5e-05
  
  comparison failed
  Obtained: 64.93314479093739
  Expected: 64.93464311280606 ± 6.5e-05
```

Reply:
Currently running `batch_processing.sh` locally -- will update once it's finished.

Reply:
Done!